plenum/server/propagator.py

```diff
@@ -142,10 +142,11 @@ def propagate(self, request: Request, clientName):
         else:
             self.requests.addPropagate(request, self.name)
             propagate = self.createPropagate(request, clientName)
-            logger.debug("{} propagating {} request {} from client {}".
-                         format(self, request.identifier, request.reqId,
-                                clientName),
-                         extra={"cli": True, "tags": ["node-propagate"]})
+            logger.info(
+                "{} propagating {} request {} from client {}".
```

Comment:
Other logs print requiest id as a tuple (client_id, request_id), could you format this one this way too? It would help to search easier

Reply:
done